extensions/lifecycle/tasks/LifecycleTask.js

```diff
@@ -909,7 +919,8 @@ class LifecycleTask extends BackbeatTask {
                     MaxUploads: MAX_KEYS,
                     KeyMarker: bucketData.details.uploadIdMarker &&
                         bucketData.details.keyMarker,
-                    UploadIdMarker: bucketData.details.uploadIdMarker,
+                    UploadIdMarker: bucketData.details.keyMarker &&
```

Comment:
Is this a bugfix? Maybe it should be reflected in the PR title/description then

Reply:
I added just in-case since both the key marker and upload id marker are required. I'll edit description to take note of the change and reason for why